models/stg_reddit_ads.yml

```diff
@@ -207,7 +207,7 @@ models:
       - name: account_id
         description: '{{ doc("account_id") }}'
       - name: ad_group_id
-        description: '{{ doc("ad_group_id") }}'
+        description: '{{ doc("ad_group_id") }}' 
```

Comment:
Should we also provide a not_null test for this field?

Reply:
udpated